File: C# ad y ad/2_ Orientado a objetos/S02.2_ Pedido/Solucion_ejercicio_pedido/Ejercicio_pedido/Pedido.cs
using System;


namespace Ejercicio_pedido
{
	class Pedido
	{
		string nombreCliente;
		string comidaPedida;
		bool seConsumeEnElLugar;


		public Pedido()
		{
			nombreCliente = "";
			comidaPedida = "";
			seConsumeEnElLugar = false;
		}


		public string GetNombreCliente()
		{
			return nombreCliente;
		}
		public string GetComidaPedida()
		{
			return comidaPedida;
		}
		public bool GetSeConsumeEnElLugar()
		{
			return seConsumeEnElLugar;
		}

		public void SetNombreCliente(string nombreCliente)
		{
			this.nombreCliente = nombreCliente;
		}
		public void SetComidaPedida(string comidaPedida)
		{
			this.comidaPedida = comidaPedida;
		}
		public void SetSeConsumeEnElLugar(bool seConsumeEnElLugar)
		{
			this.seConsumeEnElLugar = seConsumeEnElLugar;
		}
	}
}

File: C++ ad y ad/3_ Orientado a objetos/S02.2_ Pedido/Solucion_ejercicio_pedido/Ejercicio_pedido/pedido.cpp
#include "pedido.h"


Pedido::Pedido()
{
	nombreCliente = "";
	comidaPedida = "";
	seConsumeEnElLugar = false;
}
Pedido::~Pedido()
{
	cout << "El pedido de " << nombreCliente << " ha sido liberado de la memoria.\n\n";
}


string Pedido::getNombreCliente()
{
	return nombreCliente;
}
string Pedido::getComidaPedida()
{
	return comidaPedida;
}
bool Pedido::getSeConsumeEnElLugar()
{
	return seConsumeEnElLugar;
}

void Pedido::setNombreCliente(string nombreCliente)
{
	this->nombreCliente = nombreCliente;
}
void Pedido::setComidaPedida(string comidaPedida)
{
	this->comidaPedida = comidaPedida;
}
void Pedido::setSeConsumeEnElLugar(bool seConsumeEnElLugar)
{
	this->seConsumeEnElLugar = seConsumeEnElLugar;
}
